Formula/influxdb-cli.rb

```diff
@@ -0,0 +1,62 @@
+class InfluxdbCli < Formula
+  desc "CLI for managing resources in InfluxDB v2"
+  homepage "https://influxdata.com/time-series-platform/influxdb/"
+  url "https://github.com/influxdata/influx-cli.git",
+      tag:      "v2.1.0",
+      revision: "e6cad1b4aa05a801e1206496c3369246794555e0"
+  license "MIT"
+  head "https://github.com/influxdata/influx-cli.git"
+
+  livecheck do
+    url :stable
+    regex(/^v?((?!9\.9\.9)\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+  depends_on "influxdb" => :test
+
+  def install
+    ldflags = %W[
+      -s
+      -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_short_head(length: 10)}
+      -X main.date=#{time.iso8601}
+    ].join(" ")
+
+    system "go", "build", *std_go_args(ldflags: ldflags),
+           "-o", bin/"influx", "./cmd/influx"
+  end
+
+  test do
+    # Boot a test server.
+    influxd_port = free_port
+    influxd = fork do
+      exec "influxd", "--bolt-path=#{testpath}/influxd.bolt",
+                      "--engine-path=#{testpath}/engine",
+                      "--http-bind-address=:#{influxd_port}",
+                      "--log-level=error"
+    end
+    sleep 30
+
+    # Configure the CLI for the test env.
+    influx_host = "http://localhost:#{influxd_port}"
+    cli_configs_path = "#{testpath}/influx-configs"
+    ENV["INFLUX_HOST"] = influx_host
+    ENV["INFLUX_CONFIGS_PATH"] = cli_configs_path
+
+    # Check that the CLI can connect to the server.
+    assert_match "OK", shell_output("#{bin}/influx ping")
+
+    # Perform initial DB setup.
+    shell_output("#{bin}/influx setup -u usr -p fakepassword -b bkt -o org -f")
```

Comment:
```suggestion
    system "#{bin}/influx", "setup", "-u", "usr", "-p", "fakepassword", "-b", "bkt", "-o", "org", "-f"
```
Since you don't need the output here let's skip the `shell_output` call.